Makefile

```diff
@@ -44,10 +40,26 @@ tokenize-data:
 process-data:
 	python -m ${BASEDIR} --process-data
 
+.PHONY: run-all-models:
+run-all-models:
+	python -m ${BASEDIR} --logistic-regression --random-forest --xg-boost
+
 .PHONY: logistic-regression
 logistic-regression:
 	python -m ${BASEDIR} --logistic-regression
 
 .PHONY: random-forest
 random-forest:
 	python -m ${BASEDIR} --random-forest
+
+.PHONY: xg-boost
+random-forest:
+	python -m ${BASEDIR} --xg-boost
+
+.PHONY: run
+run:
+	make clean-data prepare-data run-all-models
```

Comment:
makes sense!

Reply:
This way we can keep the commands for each function, but also keep the `run` command is somewhere under 40k characters long.

![image](https://github.com/michplunkett/supreme-court-ml-predictions/assets/5885605/6f150d5d-6da3-4a4c-b983-58a232e3ee1a)